docs/setup.md

```diff
@@ -1,44 +1,61 @@
 | [Home](../README.md) |
-|--------------------------------------------|
+|----------------------|
 
 # Installation
 
 1. To install a widget, click **Content Hub** > **Discover**.
 
-2. From the list of solution pack that appears, search for and select **Continuous Delivery**. 
+2. From the list of widgets that appears, search for and select **CICD Configuration Wizard**. 
 
-3. Click the card of the **Continuous Delivery** solution pack card.
+3. Click the card of the **CICD Configuration Wizard** widget card.
 
-4. Click **Configure** on the bottom to begin configuration.
+4. Click **Install** on the bottom to begin installation.
 
 # Configuration
 
 ## CICD Configuration Wizard Widget Settings
 
 The configuration settings of the CICD Configuration widget include the following steps to setup CICD configuration:
 
-- Click the **Configure** button to bring up the following wizard screen
+1. Click the **Configure** button to bring up the following wizard screen.
 
-    ![CICD Configration](./res/cicd_configuration.png)
+    ![CICD Configuration](./res/cicd_configuration.png)
 
-- Click **Let's Get Started** to proceed
+2. Click **Let's Get Started** to proceed.
 
-- On **Define Environment** page, select the FortiSOAR environment type &mdash; **Development** and **Production**.
+3. **Define Environment**
 
-- Click **Next** on lower-right corner
+    1. Select the FortiSOAR environment type &mdash; **Development** and/or **Production**.
 
-    ![Define Environment](./res/define_environment.png)
+    2. Click **Next** on lower-right corner to proceed.
 
-- On the **Connect Version Control** page, configure your GitHub version control environment.
+        ![Define Environment](./res/define_environment.png)
 
-- Click **Next** on lower-right corner
+4. **Define Your Source Control**
```

Comment:
@cs-abhishek-shukla : Take the new screenshot of the wizard. Also, now the heading is "Select a Source Control"

Reply:
Done